### ISSUE TITLE
Why not use creator_id in confirmations_page as trade_offer_id

### ISSUE THREAD

**Issue by trgbishi** [bukson/steampy#275]
There is a creator_id in the return value of _fetch_confirmations_page, which is trade_offer_id, which is the asset_id in the confirm_sell_listing method.
That is to say, in theory, it is not necessary to query the detail information through _select_sell_listing_confirmation to obtain the asset_id. If one more creator_id is taken in _get_confirmations(), the asset_id can be obtained.
I'm just curious why it is necessary to send one more request to get the assert_id instead of directly using the creator_id in the return value of _fetch_confirmations_page

**Comment by wolfovik** [comment]
You're right. Now all the information is in the json from /getlist. But for now, I was just trying to make it work. I also want to implement a function to cancel old confirmations after a certain period of time.

**Comment by trgbishi** [comment]
> You're right. Now all the information is in the json from /getlist. But for now, I was just trying to make it work. I also want to implement a function to cancel old confirmations after a certain period of time.

In fact, in the api test a few months ago, I got the creator_id field as trade_offer_id. However, when I checked the source code of the github project created a few years ago, I found that this field was not used, so it should be available recently. I just want to make sure creator_id is safe and available, I think this issue can be closed. thank you for your explanation
